extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -704,7 +703,7 @@ export class VisibilityManagerForEmbed extends VisibilityManager {
 
   /** @override */
   getRootLayoutBox() {
-    const rootElement = dev().assertElement(this.embed.host);
+    const rootElement = dev().assertElement(this.embed.iframe);
```

Comment:
This line is the main point of the PR.

Reply:
The change makes sense. Question: is it ever possible that we want to use the embed.host layoutRect?  For example if the iframe size is greater than the embed host size. I assume it won't happen.

Reply:
I believe that is correct, but will double-check with @keithwrightbos .

Reply:
We don't want the creative to know anything about its containing element... we should treat it as if JS within a cross domain asked so I believe what's here is correct.